docs/src/reference/creating_ops.md

```diff
@@ -19,7 +19,7 @@ In order to create a source — i.e. an op with zero inputs — one should use t
 
 ## Low-level API
 
-The most general way to create an op is to create a structure that inherits from [`TimeDag.NodeOp`](@ref), and implement [`TimeDag.run_node!`](@ref).
+The most general way to create an op is to create a structure that inherits from [`TimeDag.NodeOp`](@ref), and implements [`TimeDag.run_node!`](@ref) and [`TimeDag.create_evaluation_state`](@ref).
```

Comment:
```suggestion
The most general way to create an op is to create a structure that inherits from [`TimeDag.NodeOp`](@ref), and implements both of:
* [`TimeDag.run_node!`](@ref) 
* [`TimeDag.create_evaluation_state`](@ref), OR defines [`TimeDag.stateless`](@ref) to be true.
```

Reply:
And actually I think `TimeDag.stateless` needs to be added to the list of functions for which docs should be generated. Probably immediately after `TimeDag.create_evaluation_state` in `internals.md`